autotst/job/job.py

```diff
@@ -297,31 +296,48 @@ def submit_conformer(self, conformer, restart=False):
                 nproc = 6
             else:
                 nproc = 8
+            ase_calculator.nprocshared = nproc
+            self.write_input(conformer, ase_calculator)
             if restart:
                 logging.info(
                     "Restarting calculations for {}.".format(conformer)
                 )
             else:
                 logging.info("Starting calculations for {}".format(conformer))
 
+            command = [
+                """sbatch""", 
+                """--job-name="{}" """.format(label), 
+                """--output="{}.slurm.log" """.format(label), 
+                """--error="{}.slurm.log" """.format(label),
+                """-N 1""",
+                """-n {}""".format(nproc),
+                """-t 24:00:00""",
```

Comment:
If the partition is `express`, this will violate the 1 hr time limit.  I think we should assign a `time` attribute to the calculator or job class and use that here

Reply:
Is there a way to get the maximum time allowed for a partition? Cuz if we could dynamically change the time depending on the partition, that would be ideal.